lib/oelite/util.py

```diff
@@ -57,6 +57,11 @@ def write(self, text):
             file.write(text)
         return
 
+def stracehack(msg):
```

Comment:
Believe me, when you strace oe-lite there are many many more and much larger WTF moments than meeting an occasional odd write (for example one might wonder where stat("") comes from and what good that could ever do). And having a way to isolate where the odd (or perhaps just excessively many) syscalls come from is exactly what this is for. Also, "git grep -E '[-]42'" and other semi-obvious grep-patterns immediately lead to util.py, and then an explanation is just a git blame away.